src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -714,5 +711,19 @@ public static long parseLong(final String s) {
 		}
 		return lastIndex;
 	}
+
+	// Removes the first n elements from a Collection
```

Comment:
can we make this a javadoc and have it more accurately describe what it does? it doesn't remove the first n elements, it only removes the first n elements that match the filter